api/routes/todoListRoutes.js

```diff
@@ -0,0 +1,12 @@
+module.exports = (app) => {
+  const todoList = require('../controllers/todoListController');
+
+  app.route('/todos')
+    .get(todoList.list_all_todos)
+    .post(todoList.create_todo);
+  
+  app.route('/todos/:todoId')
+    .get(todoList.read_single_todo)
+    .put(todoList.update_todo_status)
+    .delete(todoList.delete_todo);
```

Comment:
oh, interesting. I didn't know you could refer to the same route path and attach callbacks for different methods like this. very cool! would be good to mention this to students if you make them do it like this; I've only taught them the `server.get('/path', (req, res) => ...)` format.